documentation/docs/pages/operators/changelog.mdx

```diff
@@ -25,14 +25,276 @@ export const CiConfig = () => (
+
+<br />
+
+<AccordionTemplate name="Delegation size for Gateways in SGP">
+- 250k NYM - Upgrading to magura in time - 2 nodes
+- 300k NYM - Upgrading to magura + bonus for a quick patch upgrade - 102 nodes
+- No delegation - not upgrading in time - 2 nodes
+</AccordionTemplate>
+
+- `nym-node` has now implemented [IPv6 support for wireguard](https://github.com/nymtech/nym/pull/5059)
+
+- [`network_tunnel_manager.sh` updated](network): run the commands below to make sure 
+
+<br />
+
+<AccordionTemplate name={<TunnelManagerCommands/>}>
+These commands can be run one by one or copy-pasted and run as a block.
+```sh
+mkdir $HOME/nym-binaries; \
+
+curl -L https://raw.githubusercontent.com/nymtech/nym/refs/heads/develop/scripts/network_tunnel_manager.sh -o $HOME/nym-binaries/network_tunnel_manager.sh && \
+chmod +x $HOME/nym-binaries/network_tunnel_manager.sh; \
+
+$HOME/nym-binaries/network_tunnel_manager.sh check_nymtun_iptables  ; \
+$HOME/nym-binaries/network_tunnel_manager.sh remove_duplicate_rules nymtun0 ;\
+$HOME/nym-binaries/network_tunnel_manager.sh remove_duplicate_rules nymwg;\
+$HOME/nym-binaries/network_tunnel_manager.sh check_nymtun_iptables ; \
+$HOME/nym-binaries/network_tunnel_manager.sh adjust_ip_forwarding ; \
+$HOME/nym-binaries/network_tunnel_manager.sh apply_iptables_rules ; \
+$HOME/nym-binaries/network_tunnel_manager.sh check_nymtun_iptables ; \
+$HOME/nym-binaries/network_tunnel_manager.sh apply_iptables_rules_wg ; \
+$HOME/nym-binaries/network_tunnel_manager.sh configure_dns_and_icmp_wg ; \
+$HOME/nym-binaries/network_tunnel_manager.sh adjust_ip_forwarding ; \
+$HOME/nym-binaries/network_tunnel_manager.sh check_ipv6_ipv4_forwarding; \
+
+systemctl daemon-reload && service nym-node restart && journalctl -u nym-node -f 
+```
+
+Then run the jokes in a new window for control
+```sh
+$HOME/nym-binaries/network_tunnel_manager.sh joke_through_the_mixnet
+$HOME/nym-binaries/network_tunnel_manager.sh joke_through_wg_tunnel
+```
+</AccordionTemplate>
+
+#### Tools
+
+- **[New APIs documentation](../apis/introduction)** with interactive APIs generated from the OpenAPI specs of various API endpoints offered by bits of Nym infrastructure run both by Nym and community operators for both Mainnet and the Sandbox testnet.
+- [Nym Harbourmaster](https://harbourmaster.nymtech.net/) has a new tab called `CONTRACT EXPLORER` querying data from Nym mixnet contract in real time.
+- [Nym Explorer](https://explorer.nymtech.net) is updated to read migrated nodes correctly
+- [New community explorer by SpectreDAO](https://explorer.nym.spectredao.net/dashboard) offers Nym Network dashboard, Node overview and Account stats view functions for operators and delegators.
+- [`nym-vpnc`](../developers/nymvpncli) build and run documentation, for those who don't want to use the Nym VPN GUIs.
```

Comment:
ditto

Reply:
I am aware, I won't deploy it up until all s merged together